contrib/inventory/foreman.py

```diff
@@ -168,21 +191,40 @@ def _get_json(self, url, ignore_errors=None):
                 break
         return results
 
+    def _get_organization_id(self, organization_label):
+        url = "%s/katello/api/v2/organizations?search=label=%s" % (self.foreman_url, urllib.quote_plus(organization_label))
```

Comment:
Since you use requests, why don't you let that handle the escaping for you? You would need fiddle with the params dict but I think it would lead to cleaner code.